diplom.py

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/python
+import ml_metrics as metrics
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--model", choices=["random_forest", "logistic_regression"], default=["random_forest"], dest="model")
+args = parser.parse_args()
+print args.model
```

Comment:
Please write a richer logging message, e.g.: 
print "Model used in this run is %s" % args.model